scripts/prepare.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable strict */
+
+'use strict'
+
+const isCi = process.env.CI !== undefined
+if (!isCi) {
+  // eslint-disable-next-line global-require
+  require('husky').install()
```

Comment:
why is this script required?

Reply:
It is necessary to install `husky` only in local environment after `npm i` is executed, so that CI environment is not affected by it (e.g. when installing production only dependencies).

Considering previous comment, this script can be removed altogether, so that installation is performed only on user request.